src/routes/admin/profiles/deleteProfile.ts

```diff
@@ -3,33 +3,35 @@
  * SPDX-License-Identifier: Apache-2.0
  * Author : Ramu Bachala
  **********************************************************************/
+import Logger from '../../../Logger'
+import { AMTConfiguration } from '../../../models/Rcs'
 import { IProfilesDb } from '../../../repositories/interfaces/IProfilesDb'
 import { ProfilesDbFactory } from '../../../repositories/ProfilesDbFactory'
 import { PROFILE_NOT_FOUND, PROFILE_ERROR } from '../../../utils/constants'
 import { EnvReader } from '../../../utils/EnvReader'
 
 export async function deleteProfile (req, res): Promise<void> {
+  const log = new Logger('deleteProfile')
   let profilesDb: IProfilesDb = null
   const { profileName } = req.params
   try {
     profilesDb = ProfilesDbFactory.getProfilesDb()
-    const profile = await profilesDb.getProfileByName(profileName)
-    if (typeof profile === 'undefined' || profile === null) {
+    const profile: AMTConfiguration = await profilesDb.getProfileByName(profileName)
+    if (Object.keys(profile).length === 0) {
```

Comment:
I'm not sure Object.keys is the best way for this check. I think a simple "profile == null" should suffice? When there is a result, iterating over all of the keys could be a performance concern.

Reply:
done